common/spec/dependabot/clients/azure_spec.rb

```diff
@@ -181,6 +181,107 @@
     end
   end
 
+  describe "#create_pull_request" do
+    subject do
+      client.create_pull_request(
+        pr_name, source_branch, target_branch,
+        pr_description, labels, reviewers, work_item
+      )
+    end
+
+    let(:pr_name) { "test-create-pr" }
+    let(:source_branch) { "feature/test-create-pr" }
+    let(:target_branch) { "main" }
+    let(:pr_description) { "PR description" }
+    let(:labels) { [] }
+    let(:reviewers) { [] }
+    let(:work_item) { nil }
+    let(:create_pull_request_url) do
+      base_url + "/_apis/git/repositories/#{source.unscoped_repo}/pullrequests?api-version=5.0"
+    end
+
+    context "when response is 200" do
+      response_body = fixture("azure", "create_pull_request_details.json")
+
+      before do
+        @request_body = "{\"sourceRefName\":\"refs/heads/#{source_branch}\","
```

Comment:
I don't think this needs to be an instance variable, it's only referenced in this block, so we can just make it local to the block (drop the `@`)